core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+{% use "@Contao/component/_stylesheet.html.twig" %}
+
+{% block content %}
+    <div class="swiper" data-delay="{{ delay }}" data-speed="{{ speed }}" data-offset="{{ offset }}" data-loop="{{ loop }}">
```

Comment:
```suggestion
{% set attributes = attrs(attributes|default)
    .addClass('swiper')
    .set('data-delay', delay)
    .set('data-speed', speed)
    .set('data-offset', offset)
    .set('data-loop', loop)
%}

{% block content %}
```

Or is there a specific reason for using the extra nested `<div class="swiper"…>`?

Reply:
Try adding a headline to the outer element. If it still works then, we can get rid of the nested DIV.

Reply:
Yes, headline works in my test setup with the suggested changes applied.

Reply:
I don‘t know what is different on my system, but it does not work without the wrapper for me (Contao Official Demo). Maybe because the outer element has the grid classes.

So we should keep the wrapper, but we surely can optimize it regarding the attributes.

Reply:
See fdf68f42d64dcfbb1b6422ebe68ffdc9ffbb3886.